app/controllers/explain_controller.rb

```diff
@@ -28,6 +28,12 @@ def show
 
   private
 
+  def access_allowed
```

Comment:
```suggestion
  def access_allowed?
```

Suggest matching ruby convention of naming functions that return booleans as questions.